### ISSUE TITLE
Can't run in a background docker container

### ISSUE THREAD

**Issue by freman** [junegunn/redis-stat#38]
Doesn't let me run it as a background docker container without passing TERM and hackery.

```
root@command:~# docker run -d -p 7080:80 redis-stat --style=ascii --no-color redis-slave.ladbrokes.net.au:6379 10
21a8c8e5fc9d8c9a2cb1d61d6b605dd31cebde0651164a2d22f1fffa41fd2bad
root@command:~# docker logs 21
tput: No value for $TERM and no -T specified
tput: No value for $TERM and no -T specified
== Sinatra/1.3.6 has taken the stage on 80 for production with backup from Thin
>> Thin web server (v1.5.1 codename Straight Razor)
>> Maximum connections set to 1024
>> Listening on 0.0.0.0:80, CTRL+C to stop
:screen_width must be a positive integer
```

**Comment by junegunn** [comment]
Thanks for the report. I'll release the next patch version and let you know.

**Comment by junegunn** [comment]
Released 0.4.12 with the fix.
